ckan/logic/schema.py

```diff
@@ -411,6 +411,7 @@ def default_user_schema():
         'reset_key': [ignore],
         'activity_streams_email_notifications': [ignore_missing],
         'state': [ignore_missing],
+        'extras': [ignore_missing, dict]
```

Comment:
Ok, if you'd like to accept any JSON as values (that aligns with how Resource extras works and is more flexible in general, so I'm :+1: ) then something like this instead of `dict`:

```python
def json_object(value):
    "make sure value can be serialized as a json object"
    try:
        if not json.dumps(value).startswith('{'):
            raise Invalid('Not a JSON object')
    except ValueError as e:
        raise Invalid(e... )
    return value
```